### ISSUE TITLE
5.4.2.10, EXAMPLE: "EXAMPLE" should be on its own line.

### ISSUE THREAD

**Issue by ERyan71258** [SMPTE/st429-16#57]
In the example:

![image](https://github.com/SMPTE/st429-16/assets/118845840/fe464ba9-5c77-44f9-bcf4-43a779f7b418)

"EXAMPLE" should be on its own line. The em dash can also be removed. I would also indent the example text, especially in the case of multiple lines in an example, so that it is not confused with the body text. For example:

![image](https://github.com/SMPTE/st429-16/assets/118845840/3847be0d-2325-473e-86ee-2bb71f77e62e)

**Comment by SteveLLamb** [comment]
Agreed as per https://www.iso.org/sites/directives/current/part2/index.xhtml#_idTextAnchor348, will move this to tooling.
